paymentsheet/src/main/java/com/stripe/android/lpmfoundations/paymentmethod/AddPaymentMethodRequirement.kt

```diff
@@ -64,8 +63,7 @@ internal enum class AddPaymentMethodRequirement {
             val noUsBankAccount = USBankAccount.code !in paymentMethodTypes
             val supportsBankAccounts = "bank_account" in metadata.stripeIntent.linkFundingSources
             val isDeferred = metadata.stripeIntent.clientSecret == null
-            val isEnabled = FeatureFlags.instantDebits.isEnabled
-            return noUsBankAccount && supportsBankAccounts && !isDeferred && isEnabled
+            return noUsBankAccount && supportsBankAccounts && !isDeferred
```

Comment:
We no longer need a feature flag, as the `bank_account` funding source is now only included if the request comes from a supported client version (or from an explicitly-enabled merchant account).